ble/GattClient.h

```diff
@@ -281,7 +294,11 @@ class GattClient {
      * GATT server.
      */
     void onHVX(HVXCallback_t callback) {
-        onHVXCallback = callback;
+        onHVXCallbackChain.add(callback);
+    }
+
+    HVXCallbackChain_t& onHVX() { 
```

Comment:
please add a documentation header. it might be good to add a code-example showing its use